mix.exs

```diff
@@ -33,6 +33,9 @@ defmodule EChronicler.MixProject do
   # Type `mix help deps` for examples and options.
   defp deps do
     [
+      {:credo, "~> 1.6", only: [:dev, :test], runtime: false},
+      {:timex, "~> 3.0"},
+      {:tzdata, "~> 1.1"},
```

Comment:
Since `timex` uses `tzdata` you probably can remove the explicit `tzdata` dependency.